rplidar.go

```diff
@@ -51,7 +55,8 @@ type Rplidar struct {
 
 // Config describes how to configure the RPlidar component.
 type Config struct {
-	DevicePath string `json:"device_path"`
+	DevicePath   string `json:"device_path"`
+	RplidarModel string `json:"rplidar_model"`
```

Comment:
model? rplidar_model? I'm also looking into auto-detecting model but that may not be possible.

Reply:
Only way appears to be based on the device path
- (A1) `cu.usbserial-0001`
- (S1) `cu.usbserial-120`

And I'm against auto-detecting based on that as I'm not sure how uniform this is across systems and robust it is to various combinations of connected hardware

Reply:
We should have a quick scope for this config change.

Reply:
Sounds good for the user to specify the rplidar model explicitly